lib/src/reporters/console_reporter.dart

```diff
@@ -35,12 +35,15 @@ class ConsoleReporter implements Reporter {
     CodeIssueSeverity.error: AnsiPen()..red(),
   };
 
-  final _severityHumanReadable = {
+  static const _severityHumanReadable = {
     CodeIssueSeverity.style: 'Style',
     CodeIssueSeverity.warning: 'Warning',
     CodeIssueSeverity.error: 'Error',
   };
 
+  final _designIssuesColor = AnsiPen()..yellow();
+  static const _designIssues = 'Design';
```

Comment:
Why design issues have separate severity calculation? And what do you think about allowing users to configure severity for design issues?